hibike/virtual_devices.txt

```diff
@@ -0,0 +1 @@
+/dev/pts/16 /dev/pts/29
```

Comment:
I don't think this file should be part of the commit. In fact, I think it should be part of `.gitignore` instead.